packages/cli/src/commands/lockedgold/update-delegated-amount.ts

```diff
@@ -11,8 +9,8 @@ export default class UpdateDelegatedAmount extends BaseCommand {
 
   static flags = {
     ...BaseCommand.flags,
-    from: Flags.string({ ...CustomFlags.address, required: true }),
-    to: Flags.string({ ...CustomFlags.address, required: true }),
+    from: CustomFlags.address({ ...CustomFlags.address, required: true }),
```

Comment:
i think `...CustomFlags.address,` isnt needed since that is the base already

Reply:
oops, yep.